t/25-cache-service.t

```diff
@@ -308,9 +337,9 @@ subtest 'Race for same asset' => sub {
 
     my $concurrent_test = sub {
         if (!$cache_client->enqueue($asset_request)) {
+            perform_minion_jobs($t->app->minion);
             wait_for_or_bail_out { $cache_client->status($asset_request)->is_processed } 'asset';
             my $ret = $cache_client->asset_exists('localhost', $asset);
-            Devel::Cover::report() if Devel::Cover->can('report');
```

Comment:
I think this line has to be added again

Reply:
Since you're running the jobs without forking you might be fine now without it.

Reply:
I saw in the codecov report that some lines lost their coverage:
https://app.codecov.io/gh/os-autoinst/openQA/compare/4464/changes#D1L339
So I would add it back and see if that changes.
There are still forked processes in the test, right? just not the cacheservice workers?

Reply:
Yes, please ensure that t/25-cache-service.t has 100% statement coverage. It does not make sense to have lines in tests that are never or not reliably executed – with explicitly marked uncoverable exceptions of course.